flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -231,6 +231,22 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
                             "org.apache.flink.table.runtime.functions.scalar.ArrayContainsFunction")
                     .build();
 
+    public static final BuiltInFunctionDefinition ARRAY_SORT =
+            BuiltInFunctionDefinition.newBuilder()
+                    .name("ARRAY_SORT")
+                    .kind(SCALAR)
+                    .inputTypeStrategy(
+                            or(
+                                    sequence(ARRAY_COMPARABLE),
+                                    sequence(
+                                            ARRAY_COMPARABLE,
+                                            InputTypeStrategies.explicit(
```

Comment:
If we not use  InputTypeStrategies.explicit here, it will conflict with 
`import static org.apache.flink.table.types.inference.TypeStrategies.explicit;`

Reply:
You're right. Sorry, missed that.